build/build-image/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-release-approvers
```

Comment:
Same with the PR in k/org, can we keep this to a small set first, and then expand once we work out the kinks and get policies in place?